packages/headless/src/features/debug/ranking-info-parser.test.ts

```diff
@@ -0,0 +1,76 @@
+import {parseRankingInfo} from './ranking-info-parser';
```

Comment:
Tests taken, like the base of the source code for the rankin-info-parser, from the JSUI